### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by ar13pit**
Merge after tue-robotics/tue-env#470

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
 
 services:
   - docker
 
 before_install:
-  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/install-package.sh
-  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/build-package.sh
+  - openssl aes-256-cbc -K $encrypted_3959f4b55449_key -iv $encrypted_3959f4b55449_iv
+    -in ./deploy_key.enc -out ./deploy_key -d
+  - export SSH_KEY_PRIVATE=$(cat deploy_key)
```
Comment (MatthijsBurgh) [inline .travis.yml]: Is there a need to export it? You can also read it where you need it.

File: .travis.yml [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/fix/ci/ci/install-package.sh
+  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/fix/ci/ci/build-package.sh
```
Comment (MatthijsBurgh) [inline .travis.yml]: These need to be reverted before merging.

File: .travis.yml [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
   - export PACKAGE=${TRAVIS_REPO_SLUG#*/}
 
 install:
-  - bash install-package.sh --package=$PACKAGE --branch=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH} --commit=$TRAVIS_COMMIT --pullrequest=$TRAVIS_PULL_REQUEST
+  - bash install-package.sh --package=$PACKAGE --branch=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH} --commit=$TRAVIS_COMMIT --pullrequest=$TRAVIS_PULL_REQUEST -i="tuerobotics/tue-env" -s="true" -sk="$SSH_KEY_PRIVATE"
```
Comment (MatthijsBurgh) [inline .travis.yml]: Don't like the short version of the arguments, Not readable at all

Comment (ar13pit) [reply .travis.yml]: Done.

File: .travis.yml (hunk 1 above)
Comment (ar13pit) [reply .travis.yml]: Fixed.

**Comment by ar13pit** [comment]
Drop the last 2 commits before merge

**Comment by MatthijsBurgh** [comment]
Working CI: https://travis-ci.com/github/tue-robotics/hero_bringup/jobs/297950604

**Comment by MatthijsBurgh** [comment]
Lets wait fit this till new docker image is created

**Pull request merged**